vault/resource_jwt_auth_backend_role_test.go

```diff
@@ -16,7 +16,10 @@ func TestAccJWTAuthBackendRole_import(t *testing.T) {
 	backend := acctest.RandomWithPrefix("jwt")
 	role := acctest.RandomWithPrefix("test-role")
 	resource.Test(t, resource.TestCase{
-		PreCheck:     func() { testutil.TestAccPreCheck(t) },
+		PreCheck: func() {
+			testutil.TestAccPreCheck(t)
+			testutil.SkipTestEnvSet(t, testutil.EnvVarSkipVaultNext)
```

Comment:
Normally we put the `SkipTestEnv..*` at the top of the test rather than in the step's `PreCheck`. Avoids failing a test that we are going to skip.

Reply:
Sounds good. I can move it. I think I saw some env checks like that elsewhere so just copied it.

Reply:
Think you also might need to update https://github.com/hashicorp/terraform-provider-vault/blob/24044c48b6aed5d76da9d8856aa348a13a727c9b/.github/workflows/build.yml#L81

Reply:
> Sounds good. I can move it. I think I saw some env checks like that elsewhere so just copied it.

Yeah, those could be bogus depending...

Reply:
@benashz - Moved the env check out of `PreCheck` in [7020928](https://github.com/hashicorp/terraform-provider-vault/pull/1478/commits/7020928ce4ff454fe16fd13a8d0b3013312b2ca7). Also updated that `make testacc-ent ...` for CI. Let me know if that all looks okay 👍